File: Java/lab18/src/main/java/org/example/Order.java
package org.example;

public class Order {
    private String name;
    private String type;

    public Order(String name, String type) {
        this.name = name;
        this.type = type;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    @Override
    public String toString() {
        return "Order{" +
                "name='" + name + '\'' +
                ", type='" + type + '\'' +
                '}';
    }
    public Order copy(){
        return new Order(name,type);
    }
}

File: Python/lab17/Order.py
class Order:
    def __init__(self, name, type):
        self.name = name
        self.type = type

    def get_name(self):
        return self.name

    def set_name(self, name):
        self.name = name

    def get_type(self):
        return self.type

    def set_type(self, type):
        self.type = type

    def __str__(self):
        return f"Order{{name='{self.name}', type='{self.type}'}}"
